src/stravalib/model.py

```diff
@@ -972,7 +977,7 @@ class ClubActivity(strava_model.ClubActivity):
     actual return.
     """
 
-    # Spec calls for a return of a metaAthlete object (which only has id in it)
+    # Class override - spec returns metaAthlete object (which only has id in it)
```

Comment:
similarly mypy fails here. we are overriding here again for good reason. i'm not sure how to handle this. 

```
src/stravalib/model.py:981: error: Incompatible types in assignment (expression has type "Optional[ClubAthlete]", base class "ClubActivity" defined the type as "Optional[MetaAthlete]")  [assignment]
```

Reply:
perfect. i've left the note just so we (i) remember but cleaned it up a bit.